src/main/java/org/fogbowcloud/arrebol/execution/docker/DockerCommandExecutor.java

```diff
@@ -0,0 +1,57 @@
+package org.fogbowcloud.arrebol.execution.docker;
+
+import org.apache.log4j.Logger;
+import org.fogbowcloud.arrebol.execution.docker.request.ExecInstanceResult;
+import org.fogbowcloud.arrebol.execution.docker.request.WorkerDockerRequestHelper;
+
+import static java.lang.Thread.sleep;
+
+public class DockerCommandExecutor {
+
+    private static final Logger LOGGER = Logger.getLogger(DockerCommandExecutor.class);
+    private final WorkerDockerRequestHelper workerDockerRequestHelper;
+
+    public DockerCommandExecutor(WorkerDockerRequestHelper workerDockerRequestHelper) {
+        this.workerDockerRequestHelper = workerDockerRequestHelper;
+    }
+
+    public Integer executeCommand(String command, String taskId) throws Exception {
```

Comment:
`String taskId` is been used only for logging. I think it is an overkill. I'd vote for remove it.

Reply:
Removed.